src/app/components/shared/timezone/timezone.component.ts

```diff
@@ -0,0 +1,58 @@
+import { Component, Input, OnChanges } from "@angular/core";
+import { DateTimeTimezone } from "@interfaces/apiInterfaces";
+import {
+  AbstractModel,
+  isUnresolvedModel,
+  UnresolvedModel,
+} from "@models/AbstractModel";
+import { Site } from "@models/Site";
+
+@Component({
+  selector: "baw-timezone",
+  template: `
+    <span [ngbTooltip]="timezone">
+      {{ formattedDate }}
+    </span>
+  `,
+  styles: [
+    `
+      span {
+        text-decoration: underline dotted;
+        cursor: help;
+        text-decoration-skip-ink: none;
+      }
+    `,
+  ],
+})
+export class TimezoneComponent implements OnChanges {
+  @Input() public dateTime: DateTimeTimezone;
+  @Input() public site: UnresolvedModel | Site;
```

Comment:
I think this should be called `timezone` and should accept a timezone

Given how often the `site` model will be providing the timezone I'd be happy if we accepted a model as well, or as another property, but perhaps it should be decoupled and duck typed: using typescript we can accept any model with a property called `tzInfoTz`